projects/1.19/assets/ender-zoology/enderzoology/lang/zh_cn.json

```
@@ -0,0 +1,63 @@
+  "enchantment.enderzoology.decay": "衰变",
+  "enchantment.enderzoology.decay.desc": "对被攻击者施加凋零效果",
+  "enchantment.enderzoology.repellent": "斥力",
+  "enchantment.enderzoology.repellent.desc": "敌人攻击你时随机传送敌人",
+  "enchantment.enderzoology.soulbound": "灵魂绑定",
+  "enchantment.enderzoology.soulbound.desc": "死亡时保留物品，但魔咒等级可能会随机下降一级",
+  "enchantment.enderzoology.withering": "枯萎",
+  "enchantment.enderzoology.withering.desc": "对被箭击中的敌人施加凋零效果",
+  "entity.enderzoology.concussion_creeper": "震荡苦力怕",
+  "entity.enderzoology.dire_wolf": "冰原狼",
+  "entity.enderzoology.enderminy": "迷你末影人",
+  "entity.enderzoology.fallen_knight": "堕落骑士",
+  "entity.enderzoology.fallen_mount": "堕落战马",
+  "entity.enderzoology.infested_zombie": "被感染的僵尸",
+  "entity.enderzoology.owl": "猫头鹰",
+  "entity.enderzoology.owl_egg": "投掷猫头鹰蛋",
+  "entity.enderzoology.wither_cat": "凋零猫",
+  "entity.enderzoology.wither_witch": "凋零女巫",
+  "item.enderzoology.concussion_creeper_spawn_egg": "震荡苦力怕刷怪蛋",
+  "item.enderzoology.confusing_powder": "迷乱粉末",
+  "item.enderzoology.dire_wolf_spawn_egg": "冰原狼刷怪蛋",
+  "item.enderzoology.ender_fragment": "末影碎片",
+  "item.enderzoology.enderminy_spawn_egg": "迷你末影人刷怪蛋",
+  "item.enderzoology.fallen_knight_spawn_egg": "堕落骑士刷怪蛋",
+  "item.enderzoology.fallen_mount_spawn_egg": "堕落战马刷怪蛋",
+  "item.enderzoology.hunting_bow": "狩猎之弓",
+  "item.enderzoology.infested_zombie_spawn_egg": "被感染的僵尸刷怪蛋",
+  "item.enderzoology.owl_egg": "猫头鹰蛋",
+  "item.enderzoology.owl_spawn_egg": "猫头鹰刷怪蛋",
+  "item.enderzoology.wither_cat_spawn_egg": "凋零猫刷怪蛋",
+  "item.enderzoology.wither_witch_spawn_egg": "凋零女巫刷怪蛋",
+  "item.enderzoology.withering_dust": "枯萎粉尘",
+  "item.minecraft.lingering_potion.effect.confusion": "滞留型迷乱药水",
+  "item.minecraft.lingering_potion.effect.decay": "滞留型衰变药水",
+  "item.minecraft.lingering_potion.effect.displacement": "滞留型位移药水",
+  "item.minecraft.lingering_potion.effect.rising": "滞留型上浮药水",
+  "item.minecraft.potion.effect.confusion": "迷乱药水",
+  "item.minecraft.potion.effect.decay": "衰变药水",
+  "item.minecraft.potion.effect.displacement": "位移药水",
+  "item.minecraft.potion.effect.rising": "上浮药水",
+  "item.minecraft.splash_potion.effect.confusion": "喷溅型迷乱药水",
+  "item.minecraft.splash_potion.effect.decay": "喷溅型衰变药水",
+  "item.minecraft.splash_potion.effect.displacement": "喷溅型位移药水",
+  "item.minecraft.splash_potion.effect.rising": "喷溅型上浮药水",
+  "item.minecraft.tipped_arrow.effect.confusion": "迷乱之箭",
+  "item.minecraft.tipped_arrow.effect.decay": "衰变之箭",
+  "item.minecraft.tipped_arrow.effect.displacement": "位移之箭",
+  "item.minecraft.tipped_arrow.effect.rising": "上浮之箭",
+  "itemGroup.enderzoology.main": "末影动物学",
+  "subtitles.entity.dire_wolf.death": "冰原狼死亡",
+  "subtitles.entity.dire_wolf.growl": "冰原狼咆哮",
+  "subtitles.entity.dire_wolf.howl": "冰原狼嗥叫",
+  "subtitles.entity.dire_wolf.hurt": "冰原狼攻击",
+  "subtitles.entity.owl.death": "猫头鹰死亡",
+  "subtitles.entity.owl.hoot": "猫头鹰名叫",
+  "subtitles.entity.owl.hurt": "猫头鹰攻击",
+  "subtitles.entity.owl_egg.throw": "猫头鹰蛋飞出"

```

```suggestion
  "subtitles.entity.dire_wolf.death": "冰原狼：死亡",
  "subtitles.entity.dire_wolf.growl": "冰原狼：嚎叫",
  "subtitles.entity.dire_wolf.howl": "冰原狼：嗥叫",
  "subtitles.entity.dire_wolf.hurt": "冰原狼：受伤",
  "subtitles.entity.owl.death": "猫头鹰：死亡",
  "subtitles.entity.owl.hoot": "猫头鹰：鸣叫",
  "subtitles.entity.owl.hurt": "猫头鹰：受伤",
  "subtitles.entity.owl_egg.throw": "猫头鹰蛋：飞出"
```
https://minecraft.fandom.com/wiki/Subtitles
https://minecraft.fandom.com/zh/wiki/%E5%AD%97%E5%B9%95
字幕格式请参考原版字幕